app/src/main/res/values/strings.xml

```diff
@@ -858,4 +858,19 @@
     <string name="legend_noael">above dose with observed adverse effect (NOAEL)</string>
     <string name="legend_adi">above Acceptable Daily Intake (ADI)</string>
     <!-- Used for additive exposure evaluation table -->
+    <!-- Used for diets AddOn -->
+    <string name="your_diets">Your diet(s)</string>
+    <string name="diet_name">Diet name</string>
+    <string name="diets_dialog_warning_title">Warning</string>
+    <string name="warning_diets_data">Your diet(s) data stay on your device. Nothing will be upload anywhere.</string>
+    <string name="EditDiet_Name">Name</string>
+    <string name="EditDiet_Description">Description</string>
+    <string name="EditDiet_Enabled_Disabled">Enabled/Disabled</string>
+    <string name="EditDiet_Authorised_Ingredients">Authorised ingredients</string>
+    <string name="EditDiet_Hint_Ingredient_Authorised">List the authorised ingredients that should appears in green. Example : Apple, Mushroom, Tomatoe... You can complete this list latter.</string>
```

Comment:
```suggestion
    <string name="EditDiet_Hint_Ingredient_Authorised">List the authorised ingredients that should appear in green. Example : apple, mushroom, tomato… You can complete this list later.</string>
```

Reply:
The same, La prochaine fois je programme en français... quoique !